handbook/technical-overview/api-design-guide/rest-response.md

```diff
@@ -0,0 +1,89 @@
+# REST Response
+
+A REST API should use an appropriate HTTP Status Code and Content Type when responding to a client's request.
+
+## Content Type
+
+An API can support multiple response content types. It's preferred to use a JSON content by default.
+When an API supports multiple content types the client can specify the desired content type by using the
+[`Accept` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept).
+
+The following table shows a list of common content types:
+
+| Content Type                                                            | Description                                                                                    |
+| ----------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| [application/json](https://www.rfc-editor.org/rfc/rfc8259)              | **Default.** Content type for a JSON response body.                                            |
+| [application/problem+json](https://www.rfc-editor.org/rfc/rfc7807.html) | Content type for the [Problem JSON error object](errors.md) for `4xx` and `5xx` statuse codes. |
+| [application/xml](https://www.rfc-editor.org/rfc/rfc7303#section-9.1)   | Content type for a XML response body.                                                          |
+| [text/xml](https://www.rfc-editor.org/rfc/rfc7303#section-9.2)          | Same as the `application/xml` except for the type name is `'xml'`                              |
```

Comment:
I don't understand the description. 🥺

Reply:
I updated this and added a info section below on the difference. Which is from the previous RFC3023, but I like the distinction between those two. It says that `text/xml` should be used then the content is readable by casual users, while `application/xml` should be used when it is unreadable. The later RFC 7303 does not mention this difference anymore.